projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss

```diff
@@ -464,6 +489,28 @@
             map.get($items-gap, 'comfortable')
         );
 
+        @if ($variant == 'fluent') {
+            border: none;
+
+            &:focus-visible {
+                &::after {
+                    $btn-indent: rem(3px);
+                    content: '';
+                    position: absolute;
+                    top: $btn-indent;
+                    inset-inline-start: $btn-indent;
+                    pointer-events: none;
+                    width: calc(100% - (#{$btn-indent} * 2));
+                    height: calc(100% - (#{$btn-indent} * 2));
+                }
+            }
+        }
+
+        @if ($variant == 'bootstrap') {
```

Comment:
This should be deleted. I guess you've added it by mistake while resolving merge conflicts. 
See this PR https://github.com/IgniteUI/igniteui-angular/pull/13866

Reply:
ok